### ISSUE TITLE
'Accessibility mode on' not persisted when new editor opened

### ISSUE THREAD

**Issue by isidorn** [Microsoft/vscode#2301]
* Turn on 'Use tab for setting focus'
* Open a new split editor - notice that this one does not have accessibility mode on
* I would expect this mode is persisted accross editors

**Comment by bpasero** [comment]
Not sure it should really. @alexandrudima to decide, I think it should not.

**Comment by alexandrudima** [comment]
Let's discuss how we think Ctrl+M` should behave in #2304
